quadriga/src/main/java/edu/asu/diging/quadriga/core/service/SimpleUserAppService.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.diging.quadriga.core.service;
+
+import java.util.List;
+
+import edu.asu.diging.quadriga.core.exceptions.SimpleUserAppNotFoundException;
+import edu.asu.diging.quadriga.core.model.users.SimpleUserApp;
+
+public interface SimpleUserAppService {
+
+    /**
+     * Saves a mapping of the Quadriga user to client id of Citesphere apps.
+     * @param username username for the mapping
+     * @param clientId client id for the mapping
+     * @return the saved details
+     */
+    public SimpleUserApp save(String username, String clientId);
+    
+    /**
+     * Finds list of all apps that the user has access to
+     * @param username is the username to be searched
+     * @return a list of SimpleUserApps with provided username
+     */
+    public List<SimpleUserApp> findByUsername(String username);
+    
+    /**
+     * To find list of all apps that user has access to with pagination
+     * @param offset determines the starting point of the result set
+     * @param pageSize is the number of items per page
+     */
+    public List<SimpleUserApp> findByUsername(String username,int offset,int pageSize);
+    
+    /**
+     * Deletes the app with given username and app client id
+     * @param username is the username to be searched
+     * @param appClientId is the app Client Id to be searched
+     */
+    public void delete(String username, String appClientId) throws SimpleUserAppNotFoundException;
+    
+    /**
+     * 
```

Comment:
javadoc

Reply:
I didn't get which exact place needs javadoc as there is javadoc for all the methods.

Reply:
There is no description here of what this method does or what it's for.